lib/paramedic.js

```diff
@@ -106,14 +106,21 @@ ParamedicRunner.prototype = {
         }).then(function(res) {
             if (self.testsPassed) {
                 logger.info("All tests have been passed.");
-                process.exit(0);
+                return 0;
             } else {
                 logger.error("There are tests failures.");
-                process.exit(1);
+                return 1;
```

Comment:
Does the return value of this function get passed to the next promise handler, or does it need to be wrapped in `promise.resolve`?